internal/cmd/schema-registry/command_subject_update.go

```diff
@@ -38,20 +38,20 @@ func (c *command) newSubjectUpdateCommand() *cobra.Command {
 
 	addCompatibilityFlag(cmd)
 	cmd.Flags().String("compatibility-group", "", "The name for compatibility group.")
-	cmd.Flags().String("default-metadata", "", "The path to default metadata file.")
-	cmd.Flags().String("overridden-metadata", "", "The path to overridden metadata file.")
-	cmd.Flags().String("default-ruleset", "", "The path to default schema ruleset file.")
-	cmd.Flags().String("overridden-ruleset", "", "The path to overridden schema ruleset file.")
+	cmd.Flags().String("metadata-defaults", "", "The path to schema metadata defaults file.")
```

Comment:
This flag is being used for Cloud and On-Prem, so you may want to consider moving this call and `MarkFlagFilename()` to a function, i.e. `addMetadataDefaultsFlag(cmd)`